web/api/serializers.py

```diff
@@ -1,6 +1,7 @@
+from collections import defaultdict
 from dashboard.models import *
 from django.contrib.humanize.templatetags.humanize import (naturalday,
-														   naturaltime)
+														naturaltime)
```

Comment:
Something is going wrong here, see https://github.com/Security-Tools-Alliance/rengine-ng/blob/group-vulns-on-visualization/web/api/serializers.py.